### ISSUE TITLE
Product Editor: Disabled fields are not always clear

### ISSUE THREAD

**Issue by mattsherman** [woothemes/woocommerce#48951]
**Issue Description:**

It is not always clear what fields are disabled. Especially radio button fields, such as **Stock Status**...

<img width="1114" alt="Screenshot 2024-06-28 at 12 17 24" src="https://github.com/woocommerce/woocommerce/assets/2098816/c75eb583-e87b-41f2-b06c-f11ae00babb0">

It wasn't obvious to me that the radio buttons were disabled. They not look as clearly disabled as the toggle above them.

This might be more of a core Gutenberg components issue, or it might be our issue. Since I notice it right in the new product editor, I'm logged it here initially.

As far as the specific question of whether the Stock Status options should be disabled, I've opened #48952.

**Comment by j111q** [comment]
Agree, looks confusing. It should align with all our other components and have the radio button _and_ the text both be Gutenberg Grey 600 (#949494). 

> This might be more of a core Gutenberg components issue, or it might be our issue.

Not sure either, FWIW we don't seem to have a disabled state in the Core WordPress Figma file either 🤔

**Comment by amesplant** [comment]
@mattsherman cc @j111q 
I am looking into this issue, and I am not seeing the screenshot you have provided. I noticed in the other ticket that one is from the classic product editor and this screenshot is from the new product editor. What is the trick to switch between the two?

**Comment by vikrampm1** [comment]
@mattsherman @j111q can you help with the above comment so we can investigate further on this issue?

**Comment by j111q** [comment]
@amesplant @vikrampm1 You can enable the new product editor by going to Woo Settings > Advanced > Features > Experimental features > New product editor. Thanks for investigating!

**Comment by vikrampm1** [comment]
cc @Manussakis

**Comment by Manussakis** [comment]
Thanks @j111q !

I was able to activate the new Product Editor, but now I am struggling to reproduce the disabled radio inputs.

@mattsherman Could you please clarify how I can reproduce it like the screenshot you attached to the description?

Thanks!

**Comment by mattsherman** [comment]
Hi @Manussakis! I'm no longer working on the new product editor (and, also no longer working for Woo). I do not presently have a WooCommerce dev environment set up, so I can't confirm what the current state of the new product editor is, to verify whether this issue still is present.

There are a few other devs that might be able to help -- @louwie17 @mdperez86.

I hope this helps!

**Comment by vikrampm1** [comment]
As the Product Editor is being reworked, we will remove this from our active board and move it to our backlog for future consideration.

**Comment by devanshijoshi9** [comment]
Hi @nerrad 👋 

I was able to reproduce this issue in the new product editor when editing a variable product.

As mentioned earlier, if we apply the Gutenberg Grey 600 color (#949494) to both the label text and the radio button (to indicate a disabled state), this is how it appears:

<img width="731" alt="Image" src="https://github.com/user-attachments/assets/d7b228c2-ea6b-4e4c-87c4-00cee00988d0" />

------
I would like to check if this issue is still open for contribution. If so, I’m interested in working on it. Thank you!

**Comment by nerrad** [comment]
@devanshijoshi9 we won't be releasing the new product editor as a stable feature and it's in our roadmap to replace this in the future as a part of a new initiative (read [more here](https://developer.woocommerce.com/2024/11/06/whats-next-in-the-product-editor/)).

Because of that, I'm going to close this issue.
